src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -3446,7 +3381,7 @@ bool AdaptDispatch::SetCursorStyle(const DispatchTypes::CursorStyle cursorStyle)
 // - dwColor: The new RGB color value to use.
 // Return Value:
 // True if handled successfully. False otherwise.
-bool AdaptDispatch::SetCursorColor(const COLORREF cursorColor)
+void AdaptDispatch::SetCursorColor(const COLORREF cursorColor)
 {
     return SetColorTableEntry(TextColor::CURSOR_COLOR, cursorColor);
```

Comment:
Another unwanted `return`. There are several more of these calls to `SetColorTableEntry` below as well.